codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpProtocolGenerator.kt

```diff
@@ -1052,4 +1052,12 @@ private class ServerHttpProtocolImplGenerator(
             }
         }
     }
+
+    private fun getStreamingBodyTraitBounds(operationShape: OperationShape): String {
+        if (operationShape.inputShape(model).hasStreamingMember(model)) {
+            return "\n B: Into<#{SmithyHttpServer}::ByteStream>,"
```

Comment:
You can remove the `\n` and interpolate the variable in the next line.

Reply:
I'll fix the same as we decide for https://github.com/awslabs/smithy-rs/pull/1023/files#r795990687
Resolving this one in the meanwhile.